.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "C_Cpp.default.intelliSenseMode": "linux-gcc-x64"
+}
```

Comment:
please be careful with the files you add to your commits.. it should be a good opportunity to check if you are using hard tabs on your editor..